packages/components/src/components/SquigglePlayground.tsx

```diff
@@ -35,6 +35,7 @@ interface PlaygroundProps {
   /** If code is set, component becomes controlled */
   code?: string;
   onCodeChange?(expr: string): void;
+  onSettingsChange?(settings: any): void;
```

Comment:
We should type this, but doesn't seem urgent (I'd like to clean up playground props and bundle all settings in a sub-object, then it'd be easier to set a type, but that can be done in a separate PR).